contracts/src/v0.8/keystone/CapabilityRegistry.sol

```diff
@@ -107,21 +107,28 @@ contract CapabilityRegistry is OwnerIsCreator, TypeAndVersionInterface {
     bytes config;
   }
 
+  struct DONNodeCapabilityConfig {
+    /// @notice The set of p2pIds of nodes that belong to this DON. A node (the same
+    // p2pId) can belong to multiple DONs.
+    EnumerableSet.Bytes32Set nodes;
+    /// @notice The set of capabilityIds
+    EnumerableSet.Bytes32Set capabilityIds;
```

Comment:
Could these be simple arrays now?

Reply:
`capabilityIds` can as we can validate that a duplicate capability is not added using `capabilityConfigs`.  I think `nodes` has to still be an `EnumerableSet` as it'll be easier to use that to check that a node is already present in a DON compared to an array.